lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -1079,7 +1079,7 @@ void LifetimeChecker::handleInOutUse(const DIMemoryUse &Use) {
             // Normal method calls are curried, so they are:
             // (call_expr (dot_syntax_call_expr (decl_ref_expr METHOD)))
             FD = dyn_cast<FuncDecl>(DSCE->getCalledValue());
-          else
+          else if (CE->getCalledValue() != nullptr)
```

Comment:
Oooh, I like `dyn_cast_or_null`.

Does anyone know if there are other cases where `getCalledValue` could return null? Specifically, if there are cases where the other `dyn_cast<FuncDecl>` on line 1081 (the one handling `SelfApply`s) could? I'm not sure if I should change that one defensively.